sample/Aguacongas.AspNetCore.Authentication.Sample/Controllers/HomeController.cs

```diff
@@ -133,12 +141,12 @@ public async Task<IActionResult> Update(AuthenticationViewModel model)
         [Route("List")]
         public async Task<IActionResult> List()
         {
-            var schemes = await _signInManager.GetExternalAuthenticationSchemesAsync();
+            System.Collections.Generic.IEnumerable<AuthenticationScheme> schemes = await _signInManager.GetExternalAuthenticationSchemesAsync();
```

Comment:
can you simplify by adding a using System.Collections.Generic or using var please 
As you probably note. I prefer using var, it's easier to refactor.